pkg/operator/apiserver/audit/audit_policies_test.go

```diff
@@ -81,16 +86,65 @@ func TestGetAuditPolicy(t *testing.T) {
 					},
 				},
 			},
-			goldenFile: "oauth.yaml",
+			goldenFile:  "oauth.yaml",
+			errContains: "",
+		},
+		{
+			name: "multipleCustomRules",
+			config: configv1.Audit{
+				Profile: "None",
+				CustomRules: []configv1.AuditCustomRule{
+					{
+						Group:   "system:authenticated:oauth",
+						Profile: "WriteRequestBodies",
+					},
+					{
+						Group:   "system:authenticated",
+						Profile: "AllRequestBodies",
+					},
+				},
+			},
+			goldenFile:  "multipleCr.yaml",
+			errContains: "",
+		},
+		{
+			name: "unknownProfile",
+			config: configv1.Audit{
+				Profile: "InvalidString",
+			},
+			errContains: "unknown audit profile \"InvalidString\"",
+		},
+		{
+			name: "unknownCustomRulesProfile",
+			config: configv1.Audit{
+				Profile: "None",
+				CustomRules: []configv1.AuditCustomRule{
+					{
+						Group:   "InvalidGroup",
+						Profile: "InvalidProfile",
+					},
+				},
+			},
+			errContains: "unknown audit profile \"InvalidProfile\" in customRules for group \"InvalidGroup\"",
 		},
-		// TODO: add test with multiple customRules
 	}
 	for _, scenario := range scenarios {
 		t.Run(scenario.name, func(t *testing.T) {
 			// act
 			policy, err := GetAuditPolicy(scenario.config)
-			if err != nil {
-				t.Fatal(err)
+			if len(scenario.errContains) == 0 && err != nil {
+				t.Errorf("Expected no error yet recieved error")
+				return
+			}
+			if len(scenario.errContains) > 0 {
+				if err == nil {
+					t.Errorf("Expected an error yet recieved no error")
```

Comment:
nit: typo

Reply:
Thank you

Reply:
Format changed due to comment below from reviewer
https://github.com/openshift/library-go/pull/1126#discussion_r674268845

Reply:
Resolved